### PR TITLE
Documentation/api comments

### PR DESCRIPTION
**Pull request by PTKu**
#7 
- Some additions to the API documentation, minor refactoring of some methods now waiting for

### CODE REVIEW

**Review by TomKovac — CHANGES_REQUESTED** [review]
Some typos in the documentation

File: src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU [hunk 1]
```diff
@@ -173,38 +176,57 @@ METHOD PROTECTED ABSTRACT Main]]></Declaration>
         <ST><![CDATA[]]></ST>
       </Implementation>
     </Method>
-    <Method Name="Open" Id="{d862846f-7f33-47b7-aa0a-9f3ebf516821}">
+    <Method Name="OnEntry" Id="{48b3abea-36b3-0d8f-1718-3531af00fe19}">
       <Declaration><![CDATA[(*~
 	<docu>
 		<summary>
-			This method is called implicitly at the entry point of this context (<see cref="TcoContext.PlcTcoContext.Run()"/>).
+			Called on entry into context. When overriden in derived class allows to exectute an arbitraty logic prior to opeing the context cycle.
```
[the remark this answers is not in the document]
Comment (PTKu) [reply src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU]: resolved https://github.com/Inxton/TcOpen/pull/49/commits/048a51e0362bd7c592c63681e396d0123a8de887

File: src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU [hunk 2]
```diff
@@ -173,38 +176,57 @@ METHOD PROTECTED ABSTRACT Main]]></Declaration>
...
       </Implementation>
     </Method>
-    <Method Name="PostRun" Id="{cc3ded61-78cb-0c12-3071-9c305dc097e3}">
-      <Declaration><![CDATA[METHOD PROTECTED PostRun
+    <Method Name="OnExit" Id="{cc3ded61-78cb-0c12-3071-9c305dc097e3}">
+      <Declaration><![CDATA[
+(*~
+	<docu>
+		<summary>
+			Called upon exit from the context. When overriden in derived class allows to exectute an arbitraty logic after to closing the context cycle.
```
[the remark this answers is not in the document]
Comment (PTKu) [reply src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU]: resolved https://github.com/Inxton/TcOpen/pull/49/commits/048a51e0362bd7c592c63681e396d0123a8de887

File: src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU [hunk 3]
```diff
@@ -219,14 +241,48 @@ END_VAR
...
+			</remarks>
+		</summary>			
+	</docu>	
+~*)
+METHOD RtcSynchronize  : REFERENCE TO TcoContext
 VAR_INPUT
+	(*~
+		<docu>
+			<summary>
+				When called with ```true``` the syncronization mechanism executes.
```
[the remark this answers is not in the document]
Comment (PTKu) [reply src/TcoCore/src/XaeTcoCore/TcoCore/POUs/Prototypes/TcoContext/TcoContext.TcPOU]: resolved https://github.com/Inxton/TcOpen/pull/49/commits/048a51e0362bd7c592c63681e396d0123a8de887

**Pull request merged**